### ISSUE TITLE
Add support for monthly archive

### ISSUE THREAD

**Issue by hacknug** [lucagrandicelli/yearly-archive-facetwp#4]
I'm working on a site for a client that also needs a monthly filter that works pretty much the same way this plugin works for years.

Would you be interested in adding support for this feature? I can try and open a PR exploring the idea. I guess it should work similarly to how the category facet works when you set its behavior to narrow the set.

We could also support weekly archives to make it more flexible (although I don't really need it right now).

**Comment by lucagrandicelli** [comment]
Yes, this plugin was initially based on another which managed months instead of years. I think i could have a look at it and maybe restore some of the previous code and adapt it. Here it is: https://github.com/petitphp/facetwp-monthly

P.S: would you like to be added as collaborator on this project?

**Comment by hacknug** [comment]
They seem to be working well together. Only issue is `facetwp-monthly` also uses the old `wp.hooks`. Will open a PR there fixing this and another issue with the facet settings and also open a new PR here fixing the same issue with the settings.

I don't mind being a collaborator. Feel free to add me as such if you want. Will also keep contributing via PRs in case anything need to change so we can discuss it before.
